include/tfm/sdfw_psa_ipc_service.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
```

Comment:
pragma vs regular include guard, I think we usually do ifndef style

Reply:
fixed